Readme.md

```diff
@@ -36,7 +36,6 @@ We recommend to use python-virtualevn (virtualenv and virtualenv-wrapper)
 ## Features
```

Comment:
We could probably use a Disclaimer section for the branch (ie: beta branch, please test and report any issue you find), as well as a BC section for the removal of presigning loader